src/linux.rs

```diff
@@ -40,71 +39,63 @@ pub fn local_broadcast_ip() -> Result<IpAddr, Error> {
 }
 
 fn local_broadcast_impl(family: RtAddrFamily) -> Result<IpAddr, Error> {
-    let mut netlink_socket = NlSocketHandle::connect(NlFamily::Route, None, &[])
+    let (netlink_socket, _) = NlRouter::connect(NlFamily::Route, None, Groups::empty())
```

Comment:
Neli suggests to migrate to NlRouter. But I'm not sure what it means to omit the second return value.

Reply:
Omitting the second parameter removes the ability to receive information from multicast groups that you've subscribed to. I believe in this case that is not necessary so this should be okay.